buildkite/src/Command/Promotion/PromotePackages.dhall

```
@@ -50,7 +50,14 @@ let PromotePackagesSpec =
           , to_channel : DebianChannel.Type
           , source_debian_repo : DebianRepo.Type
           , target_debian_repo : DebianRepo.Type
-          , new_tags : List Text
+          , new_tags :
```

I suppose it is not required, but shouldn't we also change the corresponding defaults away from `[]` ? @dkijania

---

Actually no, because then it make is optional. not providing defaults for new_tags put a requirement on client code to always provide it. Otherwise we can have hard to debug issues when one can forget to provide tag which results that it is always the sam

---

Ok. let's let it fail, then.